SYCL/ESIMD/api/functional/value_conv.hpp

```diff
@@ -70,7 +78,8 @@ std::vector<SrcT> generate_ref_conv_data() {
                        std::is_signed_v<DstT>) {
     ref_data = details::construct_ref_data<SrcT, NumElems>(
         {-0.0, max, max_half, min, min_half});
-  } else if constexpr (std::is_signed_v<SrcT> && std::is_signed_v<DstT>) {
+  } else if constexpr (std::is_signed_v<SrcT> &&
+                       type_traits::is_sycl_signed_v<DstT>) {
```

Comment:
Is it intended? Please add comment if so
```suggestion
  } else if constexpr (type_traits::is_sycl_signed_v<SrcT> &&
                       type_traits::is_sycl_signed_v<DstT>) {
```

Reply:
I've applied this comment in this commit: 474bee149e79fe6831f55e81dae33379d914e58c